### PR TITLE
Add gameplay gif

### PR DESCRIPTION
**Pull request by alexresiga**
Addresses #20

### CODE REVIEW

**Comment by alexresiga** [comment]
@gouri-panda please have a look

File: README.md [hunk 1]
```diff
@@ -49,3 +49,5 @@ We are always here to help you out and get started. </br>
 Find an issue, then solve the issue and make a pull request to the repo. If you didn't find any helpful issues for you then raise an issue and start solving the issue.
 
 ![pig_dice_game_ss](https://user-images.githubusercontent.com/43576162/95025880-d6a0c700-06aa-11eb-863b-695c089cb0fe.png)
+
+![Alt Text](https://media.giphy.com/media/diRUrEcoCE37QsywNr/giphy.gif)
```
Comment (gouri-panda) [inline README.md]: it is so small. Please use the height and width of the app's image in the read me file.

Comment (gouri-panda) [inline README.md:53]: It is so small please use the height and width of the image in the readme file.

**Comment by alexresiga** [comment]
@gouri-panda now it is perfect

**Review by gouri-panda — APPROVED** [review]

**Pull request merged**